ts/explorer-apis.ts

```diff
@@ -108,9 +108,22 @@ const getDecimals = async (
   return decimals;
 };
 
+// Chunks the range (start..end) into subranges of size `n` (or shorter for the last chunk).
```

Comment:
I usually use the same name in the comments as the one used in the code, so i would either write

`into subranges of size 'chunkSize'`

or would change the code and rename the variable to `n`